client/src/components/add-job-post-component/addJobPosting.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import dayjs from "dayjs";
+import AddJobLead from "./index";
+import UserType from "../../prop-types/UserType";
+
+function AddJobLeadParent({ currUser }) {
+  const initialState = {
+    jobInfo: {
+      id: 0,
+      title: "",
+      employer: "",
+      location: "",
+      minCompensation: null,
+      maxCompensation: null,
+      compensationRate: "",
+      hoursPerWeek: null,
+      creationDate: dayjs(),
+      expirationDate: dayjs().add(1, "month"),
+      employmentType: "",
```

Comment:
Can you make employmentType an Array and also modify the frontend such that the user can select multiple job types? See: https://mui.com/material-ui/react-select/